mlir/include/mlir/Dialect/Vector/IR/VectorOps.h

```diff
@@ -77,6 +77,14 @@ struct VectorDim {
   int64_t dim;
   bool isScalable;
 };
+
+struct AlignmentBytes {
+  uint64_t alignment = 0;
+  AlignmentBytes() = default;
+  explicit AlignmentBytes(uint64_t alignment_) : alignment(alignment_){};
```

Comment:
llvm/mlir doesn't use trailing underscores in variable names
```suggestion
  explicit AlignmentBytes(uint64_t alignment) : alignment(alignment){};
```